src/ResourceManager/Automation/Commands.Automation/Common/AutomationClient.cs

```diff
@@ -1917,6 +1918,31 @@ private static bool IsGraphRunbook(string runbookType)
                 string.Equals(runbookType, RunbookTypeEnum.GraphPowerShellWorkflow, StringComparison.OrdinalIgnoreCase));
         }
 
+        private static string SerializeWithTimeOut(string name, object inputObject)
+        {
+            int QueryTimeOut = 30000;
```

Comment:
30s seems like a long time.  Any data showing that this length of time is normal for this cmdlet usage?

Reply:
@SteveL-MSFT  I do not have any data. Based on Get-Content + ConvertTo-Json  call timings i set to 30s. Do u have any suggestions ?

Reply:
I will set to 15s